database/dbOperations.go

```diff
@@ -0,0 +1,124 @@
+package database
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/RainGaRO/go_final_project/constants"
+	"github.com/RainGaRO/go_final_project/models"
+)
+
+func NewDbHelper(db *sql.DB) *DbHelper {
+	return &DbHelper{
+		Db: db,
+	}
+}
+
+func (d *DbHelper) AddTask(t models.Task) (string, error) {
+	query := "INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)"
+	res, err := d.Db.Exec(query, t.Date, t.Title, t.Comment, t.Repeat)
+	if err != nil {
+		return "", err
+	}
+	id, err := res.LastInsertId()
+	if err != nil {
+		return "", fmt.Errorf("error while inserting task %+v: %w", t, err)
+	}
+
+	return strconv.FormatInt(id, 10), nil
+}
+
+func (d *DbHelper) DeleteTask(taskID int) error {
+	query := "DELETE FROM scheduler WHERE id = ?"
+
+	result, err := d.Db.Exec(query, taskID)
+	if err != nil {
+		return err
+	}
+
+	value, err := result.RowsAffected()
+	if value == 0 {
+		return fmt.Errorf("не получилось удалить задачу %d так как ее нет в БД", taskID)
+	}
+
+	return err
+}
+
+func (d *DbHelper) ReadTaskById(id int) (models.Task, error) {
+	query := "SELECT id, date, title, comment, repeat FROM scheduler WHERE id=?"
+	row := d.Db.QueryRow(query, id)
+
+	var task models.Task
+	err := row.Scan(&task.ID, &task.Date, &task.Title, &task.Comment, &task.Repeat)
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
+			return models.Task{}, fmt.Errorf("задача с id %v не найдена", id)
+		}
+		log.Printf("ошибка выполнения запроса к БД: %v", err)
+		return models.Task{}, fmt.Errorf("ошибка выполнения запроса к БД: %v", err)
```

Comment:
При оборачивании `%w`, при выводе в логи `%v`
```suggestion
		return models.Task{}, fmt.Errorf("ошибка выполнения запроса к БД: %w", err)
```

Reply:
Это встречается не в одном месте